app/components/Footer/index.css

```diff
@@ -56,72 +60,92 @@
+        padding: 5px 0 5px;
+        & .bp3-icon {
+          font-size: 30px;
+          color: var(--red);
+          padding-bottom: 5px;
+          @media (max-width: 768px) {
+            position: absolute;
+            left: 140px;
+            bottom: 152px;
           }
-          & .email {
-            background-color: transparent;
+        }
+        & .email {
+          background-color: transparent;
+          border: 0;
+          box-sizing: border-box;
+          color: white;
+          font-family: "Palanquin", sans-serif;
+          font-size: 14px;
+          font-weight: 400;
+          width: 100%;
+          &:focus {
             border: 0;
-            box-sizing: border-box;
-            color: white;
-            font-family: "Palanquin", sans-serif;
-            font-size: 14px;
-            font-weight: 400;
-            width: 100%;
-            &:focus {
-              border: 0;
-              outline: 0;
-            }
+            outline: 0;
           }
-          & .email-btn {
-            background-position: 95% center;
-            background-color: transparent;
-            background-image: url("/icons/footer/arrow.svg");
-            background-repeat: no-repeat;
-            border: 0;
-            box-sizing: border-box;
-            color: white;
-            font-family: "Palanquin", sans-serif;
-            font-size: 14px;
-            font-weight: 800;
-            min-width: 90px;
-            padding-right: 25px;
-            text-transform: uppercase;
-            transition: background-position 0.1s;
-            width: 90px;
-            &:hover {
-              background-position: 100% center;
-              cursor: pointer;
-            }
+          @media (max-width: 768px) {
+            position: absolute;
+            bottom: 190px;
           }
         }
-      }
-      & .footer-logos {
-        display: flex;
-        justify-content: space-between;
-        margin-top: 25px;
-        & a {
-          display: inline-block;
-          & img {
-            height: 20px;
+        & .email-btn {
+          background-position: 90% center;
+          background-color: transparent;
+          background-image: url("/icons/footer/arrow.svg");
+          background-repeat: no-repeat;
+          border: 0;
+          box-sizing: border-box;
+          color: white;
+          font-family: "Palanquin", sans-serif;
+          font-size: 14px;
+          font-weight: 800;
+          min-width: 90px;
+          padding-right: 25px;
+          text-transform: uppercase;
+          transition: background-position 0.1s;
+          width: 90px;
+          &:hover {
+            background-position: 100% center;
+            cursor: pointer;
+          }
+          @media (max-width: 768px) {
+            display: flex;
+            margin: 0 auto;
+            justify-content: center;
           }
         }
       }
     }
   }
+  & .footer-logos {
+    background-color: #404A594D;
```

Comment:
This type of 8-digit hexadecimal color notation that includes the alpha channel is actually [not supported in IE 11](https://caniuse.com/css-rrggbbaa) 😅 

Use this notation instead (ripped right from the Figma): `rgba(64, 74, 89, 0.3);`